Documentation/Signoffs/Device_Power.md

```diff
@@ -0,0 +1,162 @@
+
+<p align="center">DC Motor Power Draw: (24 volts) * (10 amps) = 240 watts</p>
+
+
+The two stepper driver controllers have a voltage requirment between 9 and 48 volts. 24 volts will be given to this system so it can be powered by the same system. It's amprage draw is between 0-5 amps. The amperage of these controllers depends on the amount of power the actual motors need, so this system must be able to produce the maximum current so the motors can recieve the maximum current if needed. This means the total power draw of both controllers will be:
+
+<p align="center">Single Controller Power Draw: (24 volts) * (5 amps) = 120 watts</p>
+
+<p align="center">Total Controller Power Draw: (120 watts) + (120 watts) = 240 watts</p>
+
+
+This places the maximum power draw of the entire system at:
+
+<p align="center">Total Power draw of the Mechanical System: (240 watts) + (240 watts) = 480 watts</p>
+
+To acomplish the total power draw required, this system will utilize a power supply capable of supplying 600 watts of power[2]. This will give the system plenty of head room in powering the three componets as it will supply 120 more watts then is required.
+
+### Extra:
+
+The extra subsystem will be a PCB board that will be controlled by the processor system. This board will have 2 types of LEDs, and there will be 10 of each. The first type will be a green LED that will require 3 volts and 20 mAmps. It will receive 5 volts from the supply and will be limited to 3 volts using a 100 ohm resistor. There will be 10 of these so the total current draw will be:
+
+<p align="center">Total current draw of the green LEDS: (10 * 0.02) = 0.2 amps  watts</p>
+
+The second type will be a red led which will require 2.5 volts and 0.05 amps. It will also receive 5 volts from the supply which will be limited by a 50 ohm resistor. There will also be 10 of these so the total current draw will be:
+
+<p align="center">Total current draw of the red LEDS: (10 * 0.05) = 0.5 amps  watts</p>
+
+These means the total current draw and thus the total power draw of this system will be:
+
+<p align="center">Total current draw of all LEDS: 0.2 + 0.5 = 0.7 amps  watts</p>
+
+<p align="center">Total Power draw of all LEDS: 5 volts * 0.7 amps = 3.5 watts  watts</p>
+
+This system will be powered by the 5 volt 10 amp supply that will also power the processor system and relay coil. 
+
+
+### Processor:
+The processor system will be controlled by a Jetson Nano developer kit. This Jetson Nano requires a voltage of 5 volts and an amperage draw of 4 amps. The total power draw of this system will then be:
+
+<p align="center">(5 volts) * (4 amps) = 20 watts</p>
+
+To acomplish this power draw, the system will utilize the 5 volt 10 amp power supply that will also be powering the extra system and relay coil[1]. 
+
+### Emergency Stop
+
```

Comment:
The estop you chose has a normally closed contact... meaning it is closed until you press the button. What does this do to you estop logic?

Reply:
I may be thinking about this wrong, but if it is NC, then the power should be flowing into the relay, turning it on (which is good because it allows the power to go to the ME system). Then when it is pressed it will turn off the ME system. So I believe this should be correct?

Reply:
No your right, after looking at the way I have it set up, the Estop needs to be a NO switch or I need to move the relay output to be NO. I will make those changes.